apis/clusters/v1beta1/validation.go

```diff
@@ -172,7 +168,7 @@ func (cps *CloudProviderSettings) ValidateCreation() error {
 
 func validateReplicationFactor(availableReplicationFactors []int, rf int) error {
 	if rf <= 0 || rf > 300 {
-		return fmt.Errorf("replication factor must be a multiple of %v, up to a maximum value of 300",
+		return fmt.Errorf("replication factor must be a equal of %v, up to a maximum value of 300",
```

Comment:
a -> an

Reply:
may be `replication factor must be one of %v....`, wdyt?

Reply:
agree changed to ` must be one of %v`